tests/utils/types.ts

```diff
@@ -26,6 +26,7 @@ export const EMISSIONS_FLAG_BORROW_ACTIVE = 1;
 export const EMISSIONS_FLAG_LENDING_ACTIVE = 2;
 export const PERMISSIONLESS_BAD_DEBT_SETTLEMENT_FLAG = 4;
 export const FREEZE_SETTINGS = 8;
+export const CLOSE_ENABLED_FLAG = 16;
```

Comment:
Is this the same set of flags that also track whether pyth push oracle has been migrated?

Reply:
no, those are in the config (due to how they are consumed in the various risk functions)